components/Table/Table.js

```diff
@@ -95,12 +118,186 @@ export class Table {
     this.tableHeader.append(tableHeaderRow);
   }
 
+  renderDropDownMenu(menuBtn, col, cell) {
+    const dropDownMenu = document.createElement('div');
+    dropDownMenu.classList.add('drop-down__menu');
+    this.menuDropDownItems = this.menuConfig.map((menuElement) => {
+      if (!col.sortable) {
+        if (menuElement.key === this.menuElMap.SORT_BY_ASC) {
+          return (this.menuDropDownItems.innerHTML = '');
+        }
+        if (menuElement.key === this.menuElMap.SORT_BY_DESC) {
+          return (this.menuDropDownItems.innerHTML = '');
+        }
+      }
```

Comment:
I'm not following what are you trying to do here?
If this column not a sortable, why we need to do `innerHTML = ''` ?
You could just return from here but which sense of `innerHTML`?
btw, `return (this.menuDropDownItems.innerHTML = '');` looks ugly